src/um/libloaderapi.rs

```diff
@@ -13,6 +13,26 @@ use shared::minwindef::{
 };
 use um::winnt::{HANDLE, LANGID, LPCSTR, LPCWSTR, LPSTR, LPWSTR, PCWSTR, PVOID};
 
+pub const GET_MODULE_HANDLE_EX_FLAG_PIN: DWORD = 0x00000001;
+pub const GET_MODULE_HANDLE_EX_FLAG_UNCHANGED_REFCOUNT: DWORD = 0x00000002;
+pub const GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS: DWORD = 0x00000004;
+
```

Comment:
There should be no blank lines in these .rs files. Please remove both of the ones you added here.